test/passport.test.js

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
+import feathers from 'feathers/client';
+import Passport from '../src/passport';
+import auth from '../src';
+
+const validToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZXhwIjozNDc2MzkyNDgwLCJpYXQiOjE0NzYzOTI0ODAsImlzcyI6ImZlYXRoZXJzIn0.0V6NKoNszBPeIA72xWs2FDW6aPxOnHzEmskulq20uyo';
+const expiredToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjEiLCJleHAiOjE0NzYzOTI0ODAsImlhdCI6MTQ3NjM5MjQ4MCwiaXNzIjoiZmVhdGhlcnMifQ.6rzpXFqWSmNEotnWo8f-SQ2Ey4rbar3f0pQKNTHdq9A';
+
+describe('Passport', () => {
+  let passport;
+  let app;
+  let options;
+
+  before(() => {
+    options = Object.assign({}, auth.defaults);
+    app = feathers();
+    passport = new Passport(app, options);
+  });
+
+  describe.skip('getJWT', () => {
```

Comment:
Yes and no. I was being lazy because they need different setup and teardown now that they are part of the `Passport` class. That whole class needs more unit tests but I figured I would come back to them.